strings/scripts/towns/daimyn_city.string

```diff
@@ -526,6 +526,24 @@ Alright, that's what I like to\nhear! You'll really love these\lchanges.\pFor ex
 #org @gText_DaimynCityNPCHouses_PocketPCHint
 I'm working on something else that's\nreally cool, but it's too early to\lshare.\pMaybe I could be convinced, say[.] if\nyou were the champion?\pHeehee! If you manage to achieve\nthat, come back and I'll show you!
 
+#org @gText_DaimynCityNPCHouses_GivingPocketPC1
+Oh my!\pWhile I've been tinkering away, you\nwent off and became the champion!\pCongratulations!\nThat's a huge accomplishment!
+
+#org @gText_DaimynCityNPCHouses_GivingPocketPC2
+Oh! I've got just the thing.\nHang on a sec, 'kay?
+
+#org @gText_DaimynCityNPCHouses_GivingPocketPC3
+Hmm[.] Now where did I put it[.]\nAha!
```

Comment:
should end in a question mark? on Now where did I put it ??

Reply:
chatted offline - `[.]` gets interpreted as `...` in the engine at runtime, and we agreed this would make sense still